Paintroid/src/androidTest/java/org/catrobat/paintroid/test/integration/dialog/ColorDialogLocalizationTest.java

```diff
@@ -0,0 +1,515 @@
+import org.catrobat.paintroid.test.integration.BaseIntegrationTestClass;
+
+import static android.test.ViewAsserts.assertBottomAligned;
+import static android.test.ViewAsserts.assertOnScreen;
+import static android.test.ViewAsserts.assertRightAligned;
+import static android.test.ViewAsserts.assertTopAligned;
+
+/**
+ * Created by dell on 2/24/2015.
+ */
+
+
+public class ColorDialogLocalizationTest extends BaseIntegrationTestClass {
+    TextView mRedTextView;
+    SeekBar mRedSeekBar;
+    TextView mRedValueTextView;
+    TextView mGreenTextView;
+    SeekBar mGreenSeekBar;
+    TextView mGreenValueTextView;
+    TextView mBlueTextView;
+    SeekBar mBlueSeekBar;
+    TextView mBlueValueTextView;
+    TextView mAlphaTextView;
+    SeekBar mAlphaSeekBar;
+    TextView mAlphaValueTextView;
+    final int RGB_TAB_INDEX = 1;
+    float downXValue;
+    float UpXValue;
+
+    public ColorDialogLocalizationTest() throws Exception {
+        super();
+
+    }
+
+   public void testABCLanguageInterface()
+    {
+        String buttonLanguage = getActivity().getString(R.string.menu_language_settings);
+        clickOnMenuItem(buttonLanguage);
+        mSolo.sleep(500);
+        mSolo.clickOnRadioButton(0);
+        mSolo.clickOnButton(mSolo.getString(R.string.done));
+        PointF point = new PointF(mCurrentDrawingSurfaceBitmap.getWidth() / 2,
+                mCurrentDrawingSurfaceBitmap.getHeight() / 2);
+
+        mSolo.clickOnScreen(point.x, point.y);
+
+    }
+
+    public void testPreconditions()
+    {
+
+        mSolo.clickOnView(mButtonTopColor);
+        mSolo.sleep(2000);
+        TabHost tabHost = (TabHost) mSolo.getView(R.id.colorview_tabColors);
+        TabWidget colorTabWidget = tabHost.getTabWidget();
+        mSolo.clickOnView(colorTabWidget.getChildAt(RGB_TAB_INDEX), true);
+        mSolo.sleep(500);
+        assertNotNull(mSolo.searchText(mSolo.getString(R.string.color_red)));
+        assertNotNull(mSolo.searchText(mSolo.getString(R.string.color_green)));
+        assertNotNull(mSolo.searchText(mSolo.getString(R.string.color_blue)));
+        mSolo.goBack();
+    }
+
+    public void testMissingLayout()
+    {
+
+        mSolo.clickOnView(mButtonTopColor);
+        mSolo.sleep(2000);
+        TabHost tabHost = (TabHost) mSolo.getView(R.id.colorview_tabColors);
+        TabWidget colorTabWidget = tabHost.getTabWidget();
+        mSolo.clickOnView(colorTabWidget.getChildAt(RGB_TAB_INDEX), true);
+        mSolo.sleep(500);
+        assertTrue("In rgb tab the red string is Missing", mSolo.searchText(mSolo.getString(R.string.color_red)));
+        assertTrue("In rgb tab and green string is Missing",mSolo.searchText(mSolo.getString(R.string.color_green)));
+        assertTrue("In rgb tab and blue string is Missing",mSolo.searchText(mSolo.getString(R.string.color_blue)));
+        mSolo.goBack();
+
+    }
+
+
+    //the test method verify that a View is displayed correctly and the Gravity is Center-Vertical
```

Comment:
this testcase does not assert the Gravity. line 128 does it within another testcase